application-licensing-licensor/application-licensing-licensor-ui/src/main/resources/Licenses/Code/LicensesNotificationsUIX.xml

```diff
@@ -202,7 +235,7 @@
       <contentType>CSS</contentType>
     </property>
     <property>
-      <name>CSS</name>
+      <name>Invalid CSS</name>
```

Comment:
This name is confusing :)
Anyway, see the next comment, I think you should leave this CSS object for general css rules on current and future license notifications